x-pack/packages/kbn-elastic-assistant/impl/assistant/use_conversation/index.tsx

```diff
@@ -63,7 +64,8 @@ interface UseConversation {
 }
 
 export const useConversation = (): UseConversation => {
-  const { allSystemPrompts, http, toasts } = useAssistantContext();
+  const { http, toasts } = useAssistantContext();
+  const { data: allPrompts } = useFetchPrompts();
```

Comment:
nit
```suggestion
  const { data: { data: allPrompts } } = useFetchPrompts();
```

Then below we can just pass `allPrompts`. This is a pattern that happens all over, we only use the `data` property. We could consider having `useFetchPrompts` return `data.data` instead of just `data`, or just destructure it like i did above. It's confusing that sometimes `allPrompts` is `allPrompts` and other times `allPromps` is `allPrompts.data`